lib/situation_room_web/router.ex

```diff
@@ -19,9 +19,13 @@ defmodule SituationRoomWeb.Router do
 
     scope "/sites" do
       get("/", SiteController, :index)
+
+      get("/create", SiteController, :create_form)
+      post("/create", SiteController, :create)
+
```

Comment:
Can we just follow RESTful routes here and use the `resources/2` function and name our controller actions by their default REST action names?

Reply:
Yes, I was going to switch that stuff over, we talked about removing the json responses from these and if we want a json api add new controllers under /api/* I'll clean this up more.